### PR TITLE
[WIP] Use the dynamic topography postprocessor for the geoid computation.

### PR DESCRIPTION
**Pull request by ian-r-rose**
This offloads the geoid dynamic topography calculation onto the dynamic topography postprocessor.

### CODE REVIEW

**Comment by ian-r-rose** [comment]
cc @Shangxin-Liu

File: source/postprocess/geoid.cc [hunk 1]
```diff
@@ -157,51 +160,42 @@ namespace aspect
...
+      // Get a pointer to the dynamic topography postprocessor.
+      Postprocess::DynamicTopography<dim> *dynamic_topography =
+        this->template find_postprocessor<Postprocess::DynamicTopography<dim> >();
+      AssertThrow(dynamic_topography != NULL,
+                  ExcMessage("Could not find the DynamicTopography postprocessor") );
+
+      // Get the already-computed dynamic topography solution.
+      const LinearAlgebra::BlockVector topo_vector = dynamic_topography->topography_vector();
+
+      // Get a pointer to the dynamic topography postprocessor.
```
Comment (gassmoeller) [inline source/postprocess/geoid.cc]: Update Comment

**Comment by gassmoeller** [comment]
Very nice, lets see what the tester says.

**Comment by ian-r-rose** [comment]
We expect the tester to fail on the geoid test. @Shangxin-Liu needs to run a couple of tests to verify that this did not break anything.

**Pull request merged**